PR-App/PR-App/Screens/UserMedicationDetailVC.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  AddNewMedicationVC.swift
+//  PR-App
+//
+//  Created by Jakub Homik on 22/05/2020.
+//  Copyright © 2020 Jakub Homik. All rights reserved.
+//
+
+import UIKit
+
+class UserMedicationDetailVC: UIViewController {
+    
+    var medicationView = UIView()
+    var doseAndCapacityView = UIView()
+    
+    var pillNameView = PillNameView()
+    var packageCapacityView = PackageCapacityView()
+    var pillDoseView = PillDoseView()
+    
+    var dosageView = DosageView()
+    var doseProgramView = DoseProgramView()
+    var capacityPillsLeft = CapacityAndLeftPillsView()
+    
+    var medicationButtonCamera = UIButton()
+    var editButton = CustomButton(text: "Change settings")
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureMedicationView()
+        configureMedicationImage()
+        configureStackView()
+        configureDoseAndCapacityView()
+        configureStackView2()
```

Comment:
`configureStackView2` function has unclear definition. It should be more understandable.